packages/chain-adapters/src/cosmossdk/CosmosSdkBaseAdapter.ts

```diff
@@ -49,6 +49,27 @@ const transformValidator = (validator: unchained.cosmossdk.types.Validator): Val
   apr: validator.apr,
 })
 
+const parsedTxToTransaction = (parsedTx: unchained.cosmossdk.ParsedTx): Transaction => ({
+  address: parsedTx.address,
```

Comment:
why no spreadsies?

Reply:
good point, there was a time when they were more distinct, but only diff down is `value` vs `totalValue` in transfers. will update